python3/vimspector/debug_session.py

```diff
@@ -1513,12 +1513,13 @@ def _Prepare( self ):
 
   def _Launch( self ):
     def failure_handler( reason, msg ):
-      text = [
-        'Launch Failed',
-        '',
-        reason,
-        '',
-        'Use :VimspectorReset to close'
+      text = [ line for lines in [
+          'Launch Failed',
+          '',
+          reason.splitlines(),
+          '',
+          'Use :VimspectorReset to close'
+        ] for line in ( lines if isinstance( lines, list ) else [ lines ] )
```

Comment:
Maybe I can make this more readable by wrapping the 4 literal strings into `[]` and simplify the list comprehension?